src/storage/secondary/RedisStorage.cpp

```diff
@@ -89,6 +102,8 @@ RedisStorage::RedisStorage(const Url& url, const AttributeMap& attributes)
   : m_url(url),
     m_prefix("ccache"), // TODO: attribute
     m_context(nullptr),
+    m_ca_cert(parse_string_attribute(attributes, "cacert").value_or("")),
```

Comment:
I'd store the optional directly as a member. That way you avoid treating the empty string as special value.

Reply:
Storing the `const char *` that is required by the library would be another option. Then it can be passed straight through.